src/noos_inv/python.py

```diff
@@ -19,6 +19,7 @@
 class InstallType(str, enum.Enum):
     poetry = "poetry"
     pipenv = "pipenv"
+    uv = "uv"
```

Comment:
**Suggestion:** Use a more descriptive variable name for the `uv` enum value to clarify its purpose and usage within the codebase. [clarity, importance: 6]
```suggestion
    uv_package_manager = "uv"
```